src/lib/Stage.class.ts

```diff
@@ -109,13 +108,12 @@ class Stage extends EventEmitter {
       generator.on('data', quad => {
         quadsGenerated++;
         writer.addQuad(quad);
-        quadCount++;
```

Comment:
Not sure why there was both `quadsGenerated` and `quadCount`. Removing one of them.

Reply:
Is the difference maybe a grand-total v.s. a total per generator?